packages/styles/scss/components/masthead/_masthead-search.scss

```diff
@@ -13,9 +13,11 @@
   padding-left: $carbon--spacing-05;
   padding-right: $carbon--spacing-05;
   margin-left: $carbon--spacing-05;
+  margin-right: $carbon--spacing-05;
```

Comment:
@ariellalgilmore Do we want this here? When the search is active and platform link is hovered, it looks weird.

![image](https://user-images.githubusercontent.com/909118/98748781-5e53b280-2388-11eb-9f66-44328754d2de.png)

@wonilsuhibm @oliviaflory Thoughts?

Reply:
<img width="500" alt="Screen Shot 2020-11-10 at 7 54 13 PM" src="https://user-images.githubusercontent.com/20210594/98763273-83e3ba80-238e-11eb-9352-10140cae5d6b.png">

This is in the designs. When the menu items are there though it causes an additional 16px of white space between the platform and the menu items too.

Reply:
@ariellalgilmore I don't think we want to introduce an extra 16px between the platform name and the menu items. In the specs we still have 16px spacing on either side of the platform name and menu items.
<img width="435" alt="Screen Shot 2020-11-11 at 9 29 45 AM" src="https://user-images.githubusercontent.com/45692622/98823972-b6c69680-2400-11eb-95b7-d61bb045ed14.png">



However, I do see why the spec might have been confusing with the search. It looks like the original spec was trying to have the padding be even on either side of the platform, but that would mean stopping the Search at the first letter of the menu item, see below
<img width="420" alt="Screen Shot 2020-11-11 at 9 30 20 AM" src="https://user-images.githubusercontent.com/45692622/98824272-1e7ce180-2401-11eb-9cdf-0fa0d1c72f9d.png">

I'm guessing it would be easier to have the Search extend to the right of the first menu item like this so we don't have to add another 16px.
<img width="444" alt="Screen Shot 2020-11-11 at 9 37 23 AM" src="https://user-images.githubusercontent.com/45692622/98824637-89c6b380-2401-11eb-90aa-957939e5e9bc.png">

Reply:
Thanks @oliviaflory for explaining!! Just updated without the margin-right. Should look like this now
<img width="426" alt="Screen Shot 2020-11-11 at 6 45 37 AM" src="https://user-images.githubusercontent.com/20210594/98825592-84f60580-23e9-11eb-8910-0875b4a807ed.png">

Reply:
Thanks @ariellalgilmore ! I will update the spec file to be more clear.